synfig-studio/images/CMakeLists.txt

```diff
@@ -250,7 +250,7 @@ foreach (IMAGE IN ITEMS ${IMAGES})
     add_custom_command(
         TARGET build_images POST_BUILD
         COMMAND synfig ${SRC}.sif --time ${TIME}f -w ${W} -h ${H}
-        COMMAND mv ${SRC}.png ${CMAKE_CURRENT_BINARY_DIR}/${IMAGE}.png
+        COMMAND ./move.sh ${SRC}.png ${CMAKE_CURRENT_BINARY_DIR}/${IMAGE}.png; exit 0
```

Comment:
I do not understand what this changes other than ignoring errors. Anyway I think it would be better to replace this with `cmake -E rename`.